apertium_apy/handlers/translate_raw.py

```diff
@@ -23,10 +23,11 @@ def get(self):
                                       len(self.get_argument('q', strip=False)))
         if pair is not None:
             pipeline = self.get_pipeline(pair)
+            deformat = self.get_argument('deformat', default='True') != 'False'
```

Comment:
Hm, was this always truthy before?

Reply:
it used to be `default=True`, but get_argument returns a string so default should be a string (translate_raw was added for use with giellatekno's grammar checker which outputs json, I don't think it has other users)